department-of-veterans-affairs/caseflow
# Make listed attorney required

Resolves [CASEFLOW-1484](https://vajira.max.gov/browse/CASEFLOW-1484)

### Description
Fix continue to next step button being enabled at all times on Add Claimant POA page

### Acceptance Criteria
- [ ] Continue to next step button is enabled

### Testing Plan
1. Go to /intake
2. Search for an appeal and begin intake
3. Fill in Review page form, selecting 'claimant not listed'
4. Add claimant, selecting **YES** for 'Do you have a VA Form 21-22 for this claimant?'
5. Continue to next step **Add Claimant's POA**
6. Verify button is disabled
7. Add a a listed attorney e.g. Ashley Lang
8. Verify button enables
9. Remove attorney
10. Verify button is disabled again 

### User Facing Changes

 BEFORE|
 ---|

https://user-images.githubusercontent.com/68879427/117223609-e01a1b00-adc2-11eb-9c65-8a57a6282fb0.mov
AFTER|
 ---|

https://user-images.githubusercontent.com/68879427/117223633-ead4b000-adc2-11eb-8a5d-9b8cd12305cc.mov
